myStyles.css

```diff
@@ -173,10 +183,47 @@
 
   .menuTextContainer {
     display: none;
+    flex-direction: column;
+    position: fixed;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    z-index: 10;
+    background-color: rgb(69, 67, 67);
```

Comment:
- [ ] Kindly make sure the background color meets the requirement `#3C3A39` as well as the text color
<img width="1200" alt="Screenshot 2023-06-21 at 10 32 58" src="https://github.com/alexansaa/Personal_Portafolio/assets/81444338/dd38b3b8-127e-455a-aa93-1dd076d019d9">